__tests__/feature/userAuthentication.test.js

```diff
@@ -71,12 +71,17 @@ describe('user authentication', () => {
 
     awsCognito.CognitoUser.mockImplementation(MockCognitoUserAuthenticationFailure)
 
+    // we don't expect any alerts to be present yet...
+    expect(screen.queryByRole('alert')).not.toBeInTheDocument()
+
     // try to login
     fireEvent.change(screen.getByLabelText('User name'), { target: { value: 'baz.le.quux@example.edu' } })
     fireEvent.change(screen.getByLabelText('Password'), { target: { value: 'password1' } })
     fireEvent.click(screen.getByRole('button', { name: 'Login' }))
 
     // an error message should be presented
     await screen.findByText(/Incorrect username or password./)
+    // ...and an accessible alert should be present now that a failed login attempt was made (TODO: figure out querying on alert text)
+    screen.getByRole('alert')
```

Comment:
I think you want `screen.getByRole('alert', { name: 'ALERT TEXT' })` to query on alert text.

Reply:
so i'd tried a couple variations of that, and the expectation always failed when i tried to check the text in addition to the alert role.

while working on #2276 later yesterday evening, i discovered the accessibility debugger, and i went back and used it to look into this problem.  i noticed there wasn't actually a `name` property:
![Screen Shot 2020-08-28 at 2 35 35 PM](https://user-images.githubusercontent.com/7741604/91617099-16ea8800-e93c-11ea-8447-3d10d9275843.png)

later, when debugging my WIP on #2276, i noticed the use of `aria-label` in the search form, and thought i'd try applying it to the login form's error message div.  and that seemed to work:
![Screen Shot 2020-08-28 at 2 36 28 PM](https://user-images.githubusercontent.com/7741604/91617197-5ca75080-e93c-11ea-979f-6ef66abc3a31.png)

i just pushed up a commit with the failing test that i was writing/you're suggesting, about to push up a commit with the fix to the login form (hopefully).